lib/PuppeteerSharp/WaitTask.cs

```diff
@@ -171,8 +171,14 @@ internal class WaitTask : IDisposable
 
         public void Dispose()
         {
-            Dispose(true);
-            GC.SuppressFinalize(this);
```

Comment:
Since the class is now `sealed` and there is not finalizer, there's no longer a need for `GC.SuppressFinalize(this)`.